hack/kwok-perftests.sh

```diff
@@ -0,0 +1,91 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script should be executed through the makefile via `make kwok-perftest`.
+
+set -e
+
+kubectl() { KUBEVIRTCI_VERBOSE=false cluster-up/kubectl.sh "$@"; }
+
+DOCKER_TAG=${DOCKER_TAG:-devel}
+DOCKER_TAG_ALT=${DOCKER_TAG_ALT:-devel_alt}
+PROMETHEUS_PORT=${PROMETHEUS_PORT:-30007}
+
+source hack/common.sh
+source hack/config.sh
+
+_default_previous_release_registry="quay.io/kubevirt"
+
+previous_release_registry=${PREVIOUS_RELEASE_REGISTRY:-$_default_previous_release_registry}
+
+perftest_docker_prefix=${manifest_docker_prefix-${docker_prefix}}
+
+if [[ ${KUBEVIRT_PROVIDER} == os-* ]] || [[ ${KUBEVIRT_PROVIDER} =~ (okd|ocp)-* ]]; then
+    oc=${kubectl}
+fi
+
+if [ -z "$kubeconfig" ]; then
+    kubeconfig="$KUBECONFIG"
+fi
+
+echo 'KWOK Performance testing'
+export KUBEVIRT_E2E_KWOK_PERF_TEST=true
+
+echo 'Create kubevirt related kwok manifests'
+kubectl apply -k tests/performance/manifests/kwok
+
+echo 'Preparing directory for artifacts'
+export ARTIFACTS=${ARTIFACTS}/performance
+export AUDIT_CONFIG=${ARTIFACTS}/perfscale-audit-cfg.json
+export AUDIT_RESULTS=${ARTIFACTS}/perfscale-audit-results.json
+rm -rf $ARTIFACTS
+mkdir -p $ARTIFACTS
+
+export TESTS_OUT_DIR=${TESTS_OUT_DIR}
+mkdir -p ${TESTS_OUT_DIR}/performance
+
+echo 'ARTIFACTS ' ${ARTIFACTS}
+echo 'TESTS_OUT_DIR ' ${TESTS_OUT_DIR}
+
+function perftest() {
+    _out/tests/ginkgo -r -slow-spec-threshold=60s $@ _out/tests/tests.test -- ${extra_args} -kubeconfig=${kubeconfig} -container-tag=${docker_tag} -container-tag-alt=${docker_tag_alt} -container-prefix=${perftest_docker_prefix} -image-prefix-alt=${image_prefix_alt} -oc-path=${oc} -kubectl-path=${kubectl} -gocli-path=${gocli} -installed-namespace=${namespace} -previous-release-tag=${PREVIOUS_RELEASE_TAG} -previous-release-registry=${previous_release_registry} -deploy-testing-infra=${deploy_testing_infra} -config=${KUBEVIRT_DIR}/tests/default-config.json --artifacts=${ARTIFACTS}
+}
+
+function perfaudit() {
+    _out/cmd/perfscale-audit/perfscale-audit --config-file=${AUDIT_CONFIG} --results-file=${AUDIT_RESULTS}
+}
+
+if [ -n "$KUBEVIRT_E2E_FOCUS" ]; then
+    export KUBEVIRT_E2E_FOCUS="${KUBEVIRT_E2E_FOCUS}|\\[sig-performance\\]"
+else
+    export KUBEVIRT_E2E_FOCUS="\\[sig-performance\\]"
+fi
+
+additional_test_args=""
+if [ -n "$KUBEVIRT_E2E_SKIP" ]; then
+    additional_test_args="${additional_test_args} --skip=${KUBEVIRT_E2E_SKIP}"
+fi
+
+if [ -n "$KUBEVIRT_E2E_FOCUS" ]; then
+    additional_test_args="${additional_test_args} --focus=${KUBEVIRT_E2E_FOCUS}"
+fi
+
+additional_test_args="${additional_test_args} --skip-package test/performance"
```

Comment:
As a future improvement I would suggest you switch to the `--label-filter` instead of the `--focus` and `--skip` logic

Reply:
So, do you want me to modify this function to include labels https://github.com/kubevirt/kubevirt/blob/main/tests/performance/framework.go#L57 and change all the occurences. Or make changes to the current describe something like this 
`Describe("Control Plane Performance Density Testing using kwok", Label("KWOK", "sig-performance", "Serial"), Serial, func() ` and make use of --label-filter flag

Reply:
It is not my intention for this to be addressed in this PR, as it would be out of scope.
But we can introduce a set of labels (`KWOK`, `SigPerformance`) so that the results will be:
`Describe("Control Plane Performance Density Testing using kwok", SigPerformance, KWOK, Serial, func() `
Or as you proposed :)